gradle.properties

```diff
@@ -1,5 +1,5 @@
 org.gradle.caching=true
 group=org.ballerinalang
 version=1.0.2-SNAPSHOT
-ballerinaLangVersion=2.0.0-Preview3-SNAPSHOT
+ballerinaLangVersion=2.0.0-Preview3
```

Comment:
```suggestion
ballerinaLangVersion=2.0.0-Preview3-SNAPSHOT
```

Reply:
@aashikam We need to keep this SNAPSHOT. We have to remove it only in the release branch.